pkg/rancher-desktop/utils/_demo_metadata.js

```diff
@@ -6,20 +6,20 @@ export default {
       'llm',
     ],
     LatestVersion: {
-      Tag:                'v0.0.5',
-      ManifestListDigest: 'sha256:9529ccde9e8ec9eb168380aa943008441c47240668bb9fda1c4bc4a324837f25',
+      Tag:                'v0.0.6',
+      ManifestListDigest: 'sha256:fa4ee6ad7b787bc42b37cddb93019ef4556a5bb1e94b4a68a98bb180134d4359',
       Platforms:          [
         {
           OS:      'linux',
           Arch:    'amd64',
-          Size:    24198807,
-          Created: '2025-03-05T04:18:20.781961497Z',
+          Size:    24198806,
+          Created: '2025-03-06T18:10:05.984712275Z',
         },
         {
           OS:      'linux',
           Arch:    'arm64',
-          Size:    23892859,
-          Created: '2025-03-05T04:24:50.002753152Z',
+          Size:    23892858,
+          Created: '2025-03-06T18:16:37.498708394Z',
         },
       ],
       Labels: {
```

Comment:
Hmm, the `.icon` label should be set:
https://github.com/rancher-sandbox/rancher-desktop-rdx-open-webui/blob/v0.0.6/Dockerfile#L42

But that's harmless, we don't actually use that field here. (We do on `main`, but this file doesn't exist anymore.)